### PR TITLE
show experiment toggle in preview mode

### PR DESCRIPTION
**Pull request by sebastianbenz**

### CODE REVIEW

**Comment by sebastianbenz** [comment]
@kul3r4 PTAL

File: templates/css/preview.css [hunk 1]
```diff
@@ -10,11 +10,20 @@ p {
...
   background: #fff;
   box-shadow: 0 2px 2px 0 rgba(0,0,0,.14),0 3px 1px -2px rgba(0,0,0,.2),0 1px 5px 0 rgba(0,0,0,.12);
   display: block;
+  margin: 0 auto;
+  max-width: 800px;
+}
+
+#experimental-mode {
+  max-width: 800px;
+  margin: 16px auto;
```
Comment (kul3r4) [inline templates/css/preview.css]: I would have also right and left margin to stay aligned with the text below.

Comment (sebastianbenz) [inline templates/css/preview.css]: I've set the left/right margin to auto in order to align it with the article below.

Comment (kul3r4) [inline templates/css/preview.css]: I tried to run it and it doesn't add any right and left margin.

**Comment by sebastianbenz** [comment]
Fixed amp-live-list samples along the way. PTAL

**Comment by kul3r4** [comment]
LGTM

**Pull request merged**